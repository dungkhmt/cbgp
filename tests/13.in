30 30
30 52
25 4
4 17
17 15
15 3
3 6
6 24
24 21
21 7
7 11
11 25
27 20
20 1
1 18
18 8
8 5
5 22
22 27
14 13
13 19
19 16
16 9
9 23
23 14
10 30
30 12
12 28
28 10
2 29
29 26
26 2
23 22
6 7
26 30
14 27
27 13
28 14
20 4
18 3
7 5
30 9
10 2
2 12
23 28
6 8
20 13
18 5
17 3
9 19
10 12
22 4
19 1
17 1
1
0 0 17 10 32 3 28 5 42 0 33 2 36 5 32 1 11 9 17 13 33 8 18 9 17 6 22 8 30 5 10 10 29 4 29 0 6 6 26 5 38 2 29 13 22 20 36 2 31 8 15 10 23 7 21 9 16 11 13 9 
