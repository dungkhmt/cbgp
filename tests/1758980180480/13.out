(24, 3), (3, 16), (16, 14), (14, 2), (2, 5), (5, 23), (23, 20), (20, 6), (6, 10), (10, 24), (26, 19), (19, 0), (0, 17), (17, 7), (7, 4), (4, 21), (21, 26), (13, 12), (12, 18), (18, 15), (15, 8), (8, 22), (22, 13), (9, 29), (29, 11), (11, 27), (27, 9), (1, 28), (28, 25), (25, 1), (22, 21), (5, 6), (25, 29), (13, 26), (26, 12), (27, 13), (19, 3), (17, 2), (6, 4), (29, 8), (9, 1), (1, 11), (22, 27), (5, 7), (19, 12), (17, 4), (16, 2), (8, 18), (9, 11), (21, 3), (18, 0), (16, 0), 
numberIntersection=0 
minAngle=0.19739555984988044 
Var(edgeLength)=-31.441085246618343 
minDistance=0.11624763874381928 
Var(nodePos)=-425.22777777777776 
sumAngle=23.958525660416534 
sumDistance=32547.44246813741 
minEdgeLength=1.4142135623730951 
sumEdgeLength=411.4914603883959 

0.0000000000 0.1973955598 -31.4410852466 0.1162476387 -425.2277777778 23.9585256604 32547.4424681374 1.4142135624 411.4914603884

0: (44, 17),
1: (31, 44),
2: (53, 8),
3: (48, 21),
4: (69, 3),
5: (58, 6),
6: (63, 7),
7: (62, 4),
8: (26, 41),
9: (32, 39),
10: (58, 12),
11: (32, 49),
12: (39, 32),
13: (37, 40),
14: (52, 12),
15: (29, 34),
16: (49, 15),
17: (56, 2),
18: (34, 30),
19: (43, 25),
20: (66, 4),
21: (49, 27),
22: (33, 52),
23: (62, 5),
24: (53, 17),
25: (31, 41),
26: (43, 32),
27: (33, 44),
28: (30, 46),
29: (27, 42),

Time taken: 66.547 s
44 17 31 44 53 8 48 21 69 3 58 6 63 7 62 4 26 41 32 39 58 12 32 49 39 32 37 40 52 12 29 34 49 15 56 2 34 30 43 25 66 4 49 27 33 52 62 5 53 17 31 41 43 32 33 44 30 46 27 42 
0: (44, 17), 1: (31, 44), 2: (53, 8), 3: (48, 21), 4: (69, 3), 5: (58, 6), 6: (63, 7), 7: (62, 4), 8: (26, 41), 9: (32, 39), 10: (58, 12), 11: (32, 49), 12: (39, 32), 13: (37, 40), 14: (52, 12), 15: (29, 34), 16: (49, 15), 17: (56, 2), 18: (34, 30), 19: (43, 25), 20: (66, 4), 21: (49, 27), 22: (33, 52), 23: (62, 5), 24: (53, 17), 25: (31, 41), 26: (43, 32), 27: (33, 44), 28: (30, 46), 29: (27, 42), 
numberIntersection=0 minAngle=0.19739555984988044 sumAngle=23.95852566042801 minEdgeLength=1.4142135623730951 sumEdgeLength=411.49146038839245 minDistance=0.11624763874381928 sumDistance=32547.442468138273 
